backend/app/core/doctransform/service.py

```
@@ -0,0 +1,138 @@
+            job_crud.update_status(job_id, TransformationStatus.PROCESSING)
+
+            # fetch source document
+            job = job_crud.read_one(job_id)
+            logger.debug(f"Fetched job | job_id={job_id} | source_document_id={job.source_document_id}")
+            source_doc = doc_crud.read_one(job.source_document_id)
+            logger.debug(f"Fetched source document | doc_id={source_doc.id}")
+
+            # Create a simple User object for AmazonCloudStorage
+            from app.models import User
+            current_user = User(id=user_id)
+            
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)